main.go

```diff
@@ -58,6 +58,9 @@ func main() {
 	a.PUT("/rooms/:room/memberships", handlers.RoomMembershipsCreate)
 	a.DELETE("/rooms/:room/memberships", handlers.RoomMembershipsDelete)
 
+    a.PUT("/messages/:message/hearts", handlers.MessagesHeart)
```

Comment:
Nit: indentation.

Also, shouldn't the route be `/rooms/:room/messages/:message/hearts`? I realize that `/rooms/:room` is sort of superfluous at the moment, but I think it creates a cleaner delegation of concerns (and also lets us do things like add numbers to messages on a per room basis).